templates/var/lib/ansible/playbooks/scaleup.yml

```diff
@@ -16,6 +16,14 @@ cat << 'EOF' > /var/lib/os-apply-config/templates/var/lib/ansible/playbooks/scal
     shell: iptables -A DOCKER -p tcp -j ACCEPT
     when: openshift_use_flannel
 
+  - name: Set up masquerading on flannel interface
+    shell: iptables -t nat -A POSTROUTING -o {{ flannel_interface }} -j MASQUERADE
+    when: openshift_use_flannel
+
+  - name: Make iptables rule permanent
+    shell: /usr/libexec/iptables/iptables.init save
+    when: openshift_use_flannel
+ 
```

Comment:
This block looks like a copy of https://github.com/redhat-openstack/openshift-on-openstack/blob/master/templates/var/lib/ansible/playbooks/flannel.yml#L4-L14

It would probably make sense to de-duplicate it by something like:
<%#openshift_use_flannel%>
- include: /var/lib/ansible/playbooks/flannel.yml
<%/openshift_use_flannel%>

And s/- hosts: nodes/- hosts: nodes,new_nodes/ in the https://github.com/redhat-openstack/openshift-on-openstack/blob/master/templates/var/lib/ansible/playbooks/flannel.yml#L4